### DISCUSSION TITLE
GetX In Table_Calendar Package

### DISCUSSION THREAD

**Discussion by ycng95** [jonataslaw/getx#1437] [Q&A]
![image](https://user-images.githubusercontent.com/84312382/118510902-fb423e80-b763-11eb-92c8-35f1c835748f.png)
![image](https://user-images.githubusercontent.com/84312382/118510978-0eeda500-b764-11eb-98ec-c720d468a8a4.png)

An error prompted when applying GetX in table_calendar widget. Does anyone have any ideas?

**Comment by ycng95** [comment]
Solved.

**Reply by NiranjanShah** [reply]
will be gr8, if you let us know, how was it resolved, will help large community
